pkg/apis/componentconfig/types.go

```diff
@@ -63,6 +63,219 @@ const (
+	// all non-kernel processes that are not already in a container. Empty
+	// for no container. Rolling back the flag requires a reboot.
+	SystemContainer string `json:"systemContainer"`
+	// configureCBR0 enables the kublet to configure cbr0 based on
+	// Node.Spec.PodCIDR.
+	ConfigureCBR0 bool `json:"configureCbr0"`
+	// maxPods is the number of pods that can run on this Kubelet.
+	MaxPods int `json:"maxPods"`
+	// dockerExecHandlerName is the handler to use when executing a command
+	// in a container. Valid values are 'native' and 'nsenter'. Defaults to
+	// 'native'.
+	DockerExecHandlerName string `json:"dockerExecHandlerName"`
+	// The CIDR to use for pod IP addresses, only used in standalone mode.
+	// In cluster mode, this is obtained from the master.
+	PodCIDR string `json:"podCIDR"`
+	// ResolverConfig is the resolver configuration file used as the basis
+	// for the container DNS resolution configuration."), []
+	ResolverConfig string `json:"resolvConf"`
+	// cpuCFSQuota is Enable CPU CFS quota enforcement for containers that
+	// specify CPU limits
+	CPUCFSQuota bool `json:"cpuCFSQuota"`
+	// containerized should be set to true if kubelet is running in a container.
+	Containerized bool `json:"containerized"`
+	// maxOpenFiles is Number of files that can be opened by Kubelet process.
+	MaxOpenFiles uint64 `json:"maxOpenFiles"`
+	// reconcileCIDR is Reconcile node CIDR with the CIDR specified by the
+	// API server. No-op if register-node or configure-cbr0 is false.
+	ReconcileCIDR bool `json:"reconcileCIDR"`
+	// registerSchedulable tells the kubelet to register the node as
+	// schedulable. No-op if register-node is false.
+	RegisterSchedulable bool `json:"registerSchedulable"`
+	// kubeAPIQPS is the QPS to use while talking with kubernetes apiserver
+	KubeAPIQPS float32 `json:"kubeAPIQPS"`
+	// kubeAPIBurst is the burst to allow while talking with kubernetes
+	// apiserver
+	KubeAPIBurst int `json:"kubeAPIBurst"`
+	// serializeImagePulls when enabled, tells the Kubelet to pull images one
+	// at a time. We recommend *not* changing the default value on nodes that
+	// run docker daemon with version  < 1.9 or an Aufs storage backend.
+	// Issue #10959 has more details.
+	SerializeImagePulls bool `json:"serializeImagePulls"`
+	// experimentalFlannelOverlay enables experimental support for starting the
+	// kubelet with the default overlay network (flannel). Assumes flanneld
+	// is already running in client mode.
+	ExperimentalFlannelOverlay bool `json:"experimentalFlannelOverlay"`
+	// outOfDiskTransitionFrequency is duration for which the kubelet has to
+	// wait before transitioning out of out-of-disk node condition status.
+	OutOfDiskTransitionFrequency unversioned.Duration `json:"outOfDiskTransitionFrequency,omitempty"`
+	// nodeIP is IP address of the node. If set, kubelet will use this IP
+	// address for the node.
+	NodeIP string `json:"nodeIP,omitempty"`
+	// Labels to add when registering the node in the cluster.
```

Comment:
nit: should this follow the convention of the other comments

nodeLabels are the labels...